README.md

```diff
@@ -9,7 +12,7 @@ When you check out G3D, you'll need some data files from the `research` and `gam
 If you install G3D using its `.hta` installer, then this will automatically install Visual Studio Community Edition for you.
 
 ## Building instructions
-The solution file (`FirstPersonScience.sln`) can build and run using [Visual Studio](https://visualstudio.microsoft.com/vs/). 
+The solution file (`FirstPersonScience.sln`) can build and run using [Visual Studio](https://visualstudio.microsoft.com/vs/).
```

Comment:
Suggest we add "2019" to "Visual Studio" here, consider renaming the section to "Build Instructions"

Reply:
Fixed in latest push.